util/cmake/SeqAnContribs.cmake

```diff
@@ -113,5 +119,27 @@ if (WIN32)
       endif (DEFINED SEQAN_CONTRIB_BASE)
 
     endforeach ()  # all contrib versions.
-endif (WIN32)
 
+    foreach (_BOOST_VERSION 1_65_1 1_65_0 1_64_0 1_63_0 1_62_0 1_61_0 1_60_0 1_59_0 1_58_0)
```

Comment:
I think we use `find(boost)` or so. There should be a variable to set the base path for finding boost.

This should make this code completely obsolete.